Makefile

```diff
@@ -16,8 +16,7 @@ TARGET=$(TARGET_DIR)/bin/$(APP_NAME)
 IMAGE_TAG?=quay.io/openshift/origin-$(APP_NAME):latest
 export IMAGE_TAG
 MAIN_PKG=cmd/manager/main.go
-export OCP_VERSION?=4.4
-IMAGE_CLUSTER_LOGGING_OPERATOR?=registry.svc.ci.openshift.org/origin/$(VERSION):cluster-logging-operator
+export OCP_VERSION?=$(shell basename $(shell find manifests/  -maxdepth 1  -not -name manifests -type d))
```

Comment:
is `shell` necessary here since its already wrapped in `$()` ?

Reply:
:man_shrugging:  i got some strange errors without it but I couldnt find a flag to show me what it evaluated to so I dont know what the proper syntax is